packages/core-data/src/entity-provider.js

```diff
@@ -1,22 +1,7 @@
 /**
  * WordPress dependencies
  */
-import {
-	createContext,
-	useContext,
-	useCallback,
-	useMemo,
-} from '@wordpress/element';
-import { useSelect, useDispatch } from '@wordpress/data';
-import { parse, __unstableSerializeAndClean } from '@wordpress/blocks';
-
-/**
- * Internal dependencies
- */
-import { STORE_NAME } from './name';
-import { updateFootnotesFromMeta } from './footnotes';
-
-const EMPTY_ARRAY = [];
+import { createContext, useContext, useMemo } from '@wordpress/element';
 
 const EntityContext = createContext( {} );
```

Comment:
@Mamaduka @youknowriad I just wanted to call this out as there was a single `EntityContext` being used across the hooks and to move  the `useEntityId` hook into it's own file, I needed to create the same there. 

I don't see any errors/failures but wanted to be sure that doing this won't cause any unexpected outcome.

Reply:
No, that's a mistake, it should use the exact same EntityContext in all the files. You can do so by creating an `entity-context` file and importing it in all the files.

Reply:
Updated with latest commit. Thanks for confirming, it felt like that was a bad idea :)